pingrid/impl.py

```diff
@@ -529,8 +552,8 @@ def to_dash_colorscale(s: str) -> List[str]:
     return cs
 
 
-def to_dash_colorscale2(s: str) -> List[str]:
-    cm = colorscale2cv2colormap(s)
+def to_dash_colorscale2(s):
+    cm = s.cv2cm()
```

Comment:
Should be able to go straight from the pingrid RGBA representation to the dash representation without passing through the opencv representation. Interdependencies would make it harder to remove one of them in the future.

Reply:
There are now a few ColorScale methods to get from one starting point to another ending point more cleanly and independently